plugins/UsersManager/lang/en.json

```diff
@@ -79,9 +79,7 @@
         "TheLoginScreen": "The login screen",
         "ThereAreCurrentlyNRegisteredUsers": "There are currently %s registered users.",
         "TokenAuth": "API Authentication Token",
-        "TokenRegenerateConfirm": "Changing the API authentication token will invalidate the user's current token. If the user is currently logged in, they may be required to log in again. Do you really want to continue?",
-        "TokenRegenerateConfirmSelf": "Changing the API authentication token will invalidate your own token. You may be required to log in again if you proceed. Do you really want to change your authentication token?",
-        "TokenRegenerateLogoutWarning": "When you re-generate your token you will be logged out and need to log in again.",
+        "TokenRegenerateConfirmSelf": "Changing the API authentication token will invalidate your own token. If the current token is in use, you need to update all API clients with the newly generated token. Do you really want to change your authentication token?",
```

Comment:
Not sure if this changed, but do we need to change the key here since the translation content is different?

Reply:
AFAIK it is not needed anymore @sgiehl

Reply:
Not needed. Tranisfex will automatically clear all translations of this key if the content changes,